src/System/Logging/LogSink/Core.hs

```diff
@@ -0,0 +1,56 @@
+module System.Logging.LogSink.Core (
+  Config(..)
+, Format
+, Sink
+, defaultFormat
+, stdErrSink
+, sysLogSink
+, mkLogSink
+) where
+
+import           Control.Monad
+import           System.IO
+import           System.Posix.Syslog
+import           System.Logging.Facade.Types
+import           System.Logging.Facade.Sink
+
+data Config = Config {
+  configFormat :: Format
+, configSinks :: [Sink]
+}
+
+type Format = LogRecord -> String
+
+type Sink = LogLevel -> String -> IO ()
```

Comment:
Ah, I see. The level is being included, because some `Sink`s (like syslog) still need that information to log the message properly.